docs/operations/clone_api.md

```diff
@@ -0,0 +1,157 @@
+for such an object is:
+```yaml
+kind: VirtualMachineClone
+apiVersion: "clone.kubevirt.io/v1alpha1"
+metadata:
+  name: testclone
+
+spec:
+  # source & target definitions
+  source:
+    apiGroup: kubevirt.io
+    kind: VirtualMachine
+    name: vm-cirros
+  target:
+    apiGroup: kubevirt.io
+    kind: VirtualMachine
+    name: vm-clone-target
+
+  # labels & annotations definitions
+  labelFilters:
+    - "*"
+    - "!someKey/*"
+  annotationFilters:
+    - "anotherKey/*"
+
+  # other identity stripping specs:
+  newMacAddresses:
+    interfaceName: "00-11-22"
+  newSMBiosSerial: "new-serial"
+```
+
+In the next section I will go through the different settings to elaborate them.
+
+#### Source & Target
+
+The source and target indicate the source/target API group, kind and name. A few important notes:
+* Currently, the only supported kind is `VirtualMachine`, but more types are expected to be supported in the future.
+  See "future roadmap" below for more info.
+* The target name is **optional**. If unspecified, the clone controller will generate a name for the target automatically.
+
+#### Label & Annotation filters
+
+These spec fields are intended to determine which labels / annotations are being copied to the target or stripped away.
+
+The filters are a list of strings. Each string represents a key that may exist at the source. Every source key that matches
+to one of these values is being copied to the cloned target. In addition, special regular-expression-like characters can be
+used:
+
+* **Wildcard character (\*)** can be used to match to anything. Wildcard can be only used at the **end** of the filter.
+  * These filters are valid:
+    * "*"
+    * "some/key*"
+  * These filters are invalid:
+    * "some/*/key"
+    * "*/key"
+* **Negation character (!)** can be used to avoid matching certain keys. Negation can be only used at the **beginning** of a filter.
+  Note that a Negation and Wildcard can be used together.
+  * These filters are valid:
+    * "!some/key"
+    * "!some/*"
+  * These filters are invalid:
+    * "key!"
+    * "some/!key"
+
+Setting label / annotation filters is **optional**. If unset, all labels / annotations will be copied as a default.
+
+#### newMacAddresses
+
+This field is used to explicitly replace MAC addresses for certain interfaces. The field is a string to string map; the
+keys represent interface names and the values represent the new MAC address for the clone target.
+
+This field is optional. By default, all mac addresses are stripped out. This suits situations when kube-mac-pool is
+deployed in the cluster which would automatically assign the target with a fresh valid MAC address.
+
+#### newSMBiosSerial
+
+This field is used to explicitly set an SMBios serial for the target.
+
+This field is optional. By default, the target would have an empty serial.
+
+
+### Creating a VirtualMachineClone object
+
+After the clone manifest is ready, we can create it:
+```bash
+kubectl create -f clone.yaml
```

Comment:
Is there actually work to implement

`virtctl clone <vmname> [<optionalnewname>]`?

Reply:
Not yet, but there sure should be!